### DISCUSSION TITLE
Weird z-order issues with navbar and column/card?

### DISCUSSION THREAD

**Discussion by Tribalbob41** [twbs/bootstrap#38785] [Q&A]
Hey everyone,

So I've got some weird issues with z-order.  I have a fixed navbar and a card element, however the card is overtop of the navbar when I scroll.  I tried to override z-order but it didn't work; anyone aware of this?

Here's a code snippet to show the general layout:
(Had to post a screen as the code didn't work)
![image](https://github.com/twbs/bootstrap/assets/21014650/8651248f-735f-4833-9ad8-e9b97b17b86a)

**Comment by Samthomaskkkkkkklk** [comment]
I never done this beforehand what is this what do this car do?
